src/frontend/src/components/Banner/Banner.jsx

```diff
@@ -245,9 +255,8 @@ export default function Banner() {
   return (
     <React.Fragment>
       <CssBaseline />
-      <div className="heroBanner">
+      <div className="heroBanner" height="100vh">
```

Comment:
The `height` needs to be done above as a class that you apply here instead.

Reply:
Okay, thank you!